packages/uploadthing/src/sdk/index.ts

```diff
@@ -248,7 +248,7 @@ export class UTApi {
 
     // TODO: Implement filtering and pagination
     const json = await this.requestUploadThing<{
-      files: { key: string; id: string }[];
+      files: { key: string; id: string; status: string }[];
```

Comment:
This is the current union:

`"Deletion Pending" | "Failed" | "Uploaded" | "Uploading"`

Reply:
Thanks, I added it. But not sure if you like it better inline or in /types